PopupBridge/src/test/java/com/braintreepayments/popupbridge/PopupBridgeTest.java

```diff
@@ -118,19 +108,24 @@ public void newInstance_addsJavascriptInterfaceToWebView() {
 
     @Test
     public void onBrowserSwitchResult_whenNotPopupBridgeRequest_doesNotCallOnComplete() {
-        mPopupBridge.onBrowserSwitchResult(1, BrowserSwitchResult.OK, null);
+        BrowserSwitchResult result = mock(BrowserSwitchResult.class);
+        when(result.getStatus()).thenReturn(BrowserSwitchResult.STATUS_OK);
+        mPopupBridge.onBrowserSwitchResult(1, result, null);
```

Comment:
Just curious - how come we need to use a mock instead of just passing `BrowserSwitchResult.OK`?

Reply:
Yeah, we made the constructor for `BrowserSwitchResult` package-private to discourage merchants from instantiating one. If PopupBridge were in the same package we would be able to use the real constructor here.

It might make sense for v4 to simplify the package structure so that we can share braintree code between sdk-agnostic libraries like `browser-switch` and `popup-bridge`. That would eliminate the need for mocking in the future.